tests/optimisation/tests.cmake

```diff
@@ -11,4 +11,4 @@ SET(${CURRENT_TEST}_SRC
     )
 
 SET(OPTIMISATION_CUBE_TRICUBIC_LAGRANGE_RESOURCE "${CMAKE_CURRENT_LIST_DIR}/tricubic.exfile")
-SET(OPTIMISATION_FIT_LINE_TIME_EX3_RESOURCE "${CMAKE_CURRENT_LIST_DIR}/data/fit_line_time.exf")
+SET("data/fit_line_time.exf" "${CMAKE_CURRENT_LIST_DIR}/data/fit_line_time.exf")
```

Comment:
The last line is probably an accidental replace, but are all these lines redundant now (in all test.cmake files)?